### DISCUSSION TITLE
Support for BRAW files

### DISCUSSION THREAD

**Discussion by bluszcz** [spacedriveapp/spacedrive#2769] [Feature Request]
It would be great that Spacedrive would be able to generate thumbs from BRAW files (video raw files from Blackmagic Design cameras), so app could be used by videographers. 

URL for BRAW sdk: https://www.blackmagicdesign.com/developer/products/braw/sdk-and-software

**Comment by HeavenVolkoff** [comment]
Hello @bluszcz, thanks for the suggestion. Unfortunately, we can't use BRAW sdk directly due to its license being incompatible with ours. Maybe, whenever we add plugin support, this feature could be added as a plugin. Or if there are any reversed engineered OSS decoders for BRAW we could implement them.
